Update use-the-system-health-session.md

sqlworldwide opened the pull request:
The system_health Extended Events session keeps ten files with a maximum size of 100 MB. You can confirm this by looking at GUI properties or scripting the definition.

`ADD TARGET package0.event_file(SET filename=N'system_health.xel',max_file_size=(100),max_rollover_files=(10)),`

sqlworldwide committed:
Update use-the-system-health-session.md

The system_health Extended Events session keeps ten files with a maximum size of 100 MB. You can confirm this by looking at GUI properties or scripting the definition.

ADD TARGET package0.event_file(SET filename=N'system_health.xel',max_file_size=(100),max_rollover_files=(10)),

Court72 commented:
@WilliamDAssafMSFT

Can you review the proposed changes? 
	
**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 
	
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

WilliamDAssafMSFT committed:
Update use-the-system-health-session.md

Hi @sqlworldwide - after reviewing the source code, this is a little more complicated. I'm going to update the PR like this, then wait for some engineering confirmation.

rwestMSFT commented:
@sqlworldwide Does it make sense to add this line, or just replace the currently noted limits? I'm not familiar with this subject matter.

@WilliamDAssafMSFT

WilliamDAssafMSFT commented:
@sqlworldwide @rwestMSFT after reviewing the source code, this is a little more complicated. It is edition dependent. I updated the PR accordingly, then I'll wait for some engineering confirmation.

dimitri-furman commented on docs/relational-databases/extended-events/use-the-system-health-session.md:
```diff
@@ -47,7 +47,7 @@ The session collects information that includes the following information:
 ## View the system_health session data
 
-The session uses both the ring buffer target and event file target to store the data. The event file target is configured with a maximum size of 5 MB and a file retention policy of 4 files.
+The session uses both the ring buffer target and event file target to store the data. The event file target is configured with a maximum size of 5 MB and a file retention policy of 4 files. In SQL Server Standard or Enterprise editions, the event file target has increased limits: a maximum size of 100 MB and a file retention policy of 10 files.
```
```suggestion
The session uses both the ring buffer target and event file target to store the data. The event file target is configured with a maximum size of 5 MB and a file retention policy of 4 files. In SQL Server Standard and Enterprise editions, the event file target has increased limits: a maximum size of 100 MB and a file retention policy of 10 files.
```

dimitri-furman reviewed (approved):
Thanks for the fix @sqlworldwide. Our apologies that this took a while to process.

WilliamDAssafMSFT committed:
Update docs/relational-databases/extended-events/use-the-system-health-session.md

Co-authored-by: Dimitri Furman <dfurman@microsoft.com>

WilliamDAssafMSFT committed:
Update use-the-system-health-session.md

WilliamDAssafMSFT commented:
#sign-off

WilliamDAssafMSFT committed:
Update use-the-system-health-session.md

WilliamDAssafMSFT commented:
#sign-off
